python/ray/tune/examples/mnist_pytorch.py

```diff
@@ -129,7 +129,7 @@ def train_mnist(config):
     args, _ = parser.parse_known_args()
 
     if args.server_address:
-        ray.util.connect(args.server_address)
+        ray.init(f"ray://{args.server_address}")
```

Comment:
actually ideally all of these examples could just use `ray.init(args.address)` right?

cc @amogkam

Reply:
Yep! The examples don't need a distinction between `args.address` and `args.server_address` now that everything is using the `ray.init()` API.

Reply:
(though @ckw017 i think you can just open an issue, we can take this action on our team)